explorer/app/models/responses.ts

```diff
@@ -95,10 +123,15 @@ export interface FileListResponse extends PaginatedResponse {
   hits: FileResponse[];
 }
 
+export interface AnvilFileListResponse extends PaginatedResponse {
+  hits: AnvilFileResponse[];
+}
+
 export type DetailResponseType =
   | ProjectResponse
   | FileResponse
-  | SampleResponse;
+  | SampleResponse
+  | AnvilFileResponse;
```

Comment:
Do we need to add `AnvilFileResponse` here if we don't have a detail page for AnVIL files?